test/padding.jl

```diff
@@ -99,5 +99,53 @@ end
   @test pad_reflect(x, (2, 2, 2, 2), dims=(1,3)) ≈
           pad_reflect(x, 2, dims=(1,3))
       
-  gradtest(x -> pad_repeat(x, (2,2,2,2)), rand(2,2,2))
+  # pad_reflect needs larger test input as padding must 
+  # be strictly less than array size in that dimension
+  gradtest(x -> pad_reflect(x, (2,2,2,2)), rand(3,3,3))
+end
+
+@testset "padding symmetric" begin
+  y = pad_symmetric(reshape(1:9, 3, 3), (2,2), dims=2)
+  @test y ==  [ 4  1  1  4  7  7  4
```

Comment:
Some strangeness with formatting in this file

Reply:
Is it the indentation/alignment? Or structuring of the test?

Reply:
The former. I see existing tests for other padding functions are also poorly formatted, but if we fix these ones we can copy the formatting here to those later.